test/test_ops.py

```diff
@@ -1450,5 +1451,25 @@ def test_is_leaf_node(self, dim, p, block_size, inplace):
         assert len(graph_node_names[0]) == 1 + op_obj.n_inputs
 
 
+class TestCIOULoss:
+    box1 = ([-1, -1, 1, 1],[-1, -1, 1, 1],[-1, -1, 1, 1],[-1, -1, 1, 1])
```

Comment:
hmm, to be on safer side can you add `dtype=torch.float`. It might just take these as ints
also 
```suggestion
    box1 = ([-1., -1., 1., 1.],[-1., -1., 1., 1.],[-1., -1., 1., 1.],[-1., -1., 1., 1.])
```

Reply:
oh yeah, will add tests with different datatypes.